packages/brand-shape/src/Blob.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { css } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import { fontFamilies } from '@leafygreen-ui/tokens';
+import { BlobProps } from './types';
+import { generateBlobPath } from './generateBlobPath';
+import { useMemo } from 'react';
+
+export function Blob({
+  shape,
+  fill = palette.green.base,
+  mode = 'production',
+  onGridCircleClick,
+  className,
+  ...rest
+}: BlobProps) {
+  const path = useMemo(() => generateBlobPath(shape), [shape]);
+
+  return (
+    <svg
+      viewBox="0 0 8 8"
+      width="100%"
+      height="100%"
+      className={className}
+      {...rest}
+    >
+      {/* DEBUG */}
```

Comment:
😂

Reply:
If I were to review this component for production purposes, I would propose that we include this logic in a `BlobDebugger` wrapper component that functions similarly to the V11Table's V11Adapter component where these svg elements can be injected into the regular blob component.